target/linux/octeon/base-files/lib/preinit/01_sysinfo

```diff
@@ -16,7 +16,9 @@ do_sysinfo_octeon() {
 	"UBNT_E220"*)
 		name="erpro"
 		;;
-
+	"ITUS_SHIELD"*)
+		name="itus_shield-router"
```

Comment:
This one should be with _comma_, the define/Device definition should be with _underscore_.
And this will only work if you implement the wildcard approach for the upgrade.